cpp/include/patterns/vertex_frontier.cuh

```diff
@@ -147,13 +147,17 @@ template <typename vertex_t, bool is_multi_gpu = false>
 class Bucket {
  public:
   Bucket(raft::handle_t const& handle, size_t capacity)
-    : handle_ptr_(&handle), elements_(capacity, invalid_vertex_id<vertex_t>::value)
+    : handle_ptr_(&handle), elements_(capacity, handle.get_stream())
   {
+    thrust::fill(rmm::exec_policy(handle_ptr_->get_stream())->on(handle_ptr_->get_stream()),
```

Comment:
Consider replacing 
`thrust::fill(rmm::exec_policy(handle_ptr_->get_stream())->on(handle_ptr_->get_stream()),`
by 
`thrust::fill(rmm::exec_policy(handle.get_stream_view()),`

This applies to all thrust calls

Reply:
Yes, same story here. This should be replaced all at once.